bettercli/decorators.py

```
@@ -23,14 +23,14 @@ def decorator(cmd:'Command'):
 
 
 @t.overload 
-def kw_option(name: 'str', type_: 'type[T]', *keys: str, default: 'T' = None, length: 'int' = 2): ...
+def kw_option(name: 'str', type: 'type[T]', *keys: str, default: 'T' = None, length: 'int' = 2): ...
 
 @t.overload
-def kw_option(name: 'str', type_: 'list[t.Optional[type]]' = [], *keys: str, default: 'list[t.Optional[t.Any]]' = [], min_length: 'int' = 2, max_length: 'int' = 2): ...
+def kw_option(name: 'str', type: 'list[t.Optional[type]]' = [], *keys: str, default: 'list[t.Optional[t.Any]]' = [], min_length: 'int' = 2, max_length: 'int' = 2): ...
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

(B006)